### DISCUSSION TITLE
Quarto OJS offline

### DISCUSSION THREAD

**Discussion by neocarto** [quarto-dev/quarto-cli#9266] [Q&A]
### Description

Is it possible tu use Quarto with ojs cells offline?
The system depends on js libraries. Is it possible to use local files? How?

**Comment by mcanouil** [comment]
Duplicate of #5971

- https://github.com/quarto-dev/quarto-cli/discussions/5971

Since the embedded version cannot be offline as already answered, the only possible alternative (don't know if Observable allows it) is to look at Observable documentation about offline and use all that in Quarto using `include-` YAML key and/or raw blocks of code.
